src/mono/wasm/debugger/tests/debugger-test/debugger-test.csproj

```diff
@@ -8,7 +8,6 @@
     <WasmGenerateAppBundle>true</WasmGenerateAppBundle>
     <OutputType>library</OutputType>
     <WasmEmitSymbolMap>true</WasmEmitSymbolMap>
-    <_WasmPThreadPoolSize Condition="'$(MonoWasmBuildVariant)' == 'multithread'">10</_WasmPThreadPoolSize>
```

Comment:
This caused failures in MT debugger tests., we should not merge without blocking them.

Reply:
My experience is that they are broken/fragile.
I set the `shouldContinueOnError: true` for `DebuggerTests_MultiThreaded` so that it would not fail the build, it would rather only become orange in AzDo.

cc @thaystg